src/Commands/ConfigTrait.php

```diff
@@ -26,6 +26,7 @@ protected function configurePPMOptions(Command $command)
             ->addOption('static-directory', null, InputOption::VALUE_REQUIRED, 'Static files root directory, if not provided static files will not be served', '')
             ->addOption('max-requests', null, InputOption::VALUE_REQUIRED, 'Max requests per worker until it will be restarted', 1000)
             ->addOption('max-execution-time', null, InputOption::VALUE_REQUIRED, 'Maximum amount of time a request is allowed to execute before shutting down', 30)
+            ->addOption('memory-limit', null, InputOption::VALUE_REQUIRED, 'Maximum amount of memory a worker is allowed to consume (in MB) before shutting down', 256)
```

Comment:
Shouldn't the default be unbound, i.e. only according to php ini settings? Not sure what the expected behaviour would be.
At least, unlimited (0? -1?) should be possible.

Reply:
I changed the default to -1. I don't think it needs to follow the php ini settings, because technically it won't kill the worker in the middle of the request to prevent it from exceeding the limit, it will just restart it after the reques is finished.